native-schema-registry/csharp/AWSGsrSerDe/AWSGsrSerDe.Tests/User.cs

```diff
@@ -0,0 +1,41 @@
+using Avro;
+using Avro.Specific;
+
+namespace AWSGsrSerDe.Tests
+{
+    public class User : ISpecificRecord
+    {
+        
+        private static readonly Schema UserSchema = Schema.Parse("{\"type\":\"record\",\"name\":\"User\",\"namespace\":\"AWSGsrSerDe.Tests\",\"fields\":[{\"name\":\"name\",\"type\":\"string\"},{\"name\":\"favorite_number\",\"type\":[\"int\",\"null\"]},{\"name\":\"favorite_color\",\"type\":[\"string\",\"null\"]}]}");
+
+        public string Name;
+        public int FavoriteNumber;
+        public string FavoriteColor;
+
+
+        public object Get(int fieldPos)
+        {
+            return fieldPos switch
+            {
+                0 => Name,
+                1 => FavoriteNumber,
+                2 => FavoriteColor,
+                _ => throw new AvroRuntimeException("Bad index " + fieldPos + " in Get()")
+            };
+            ;
+        }
+
+        public void Put(int fieldPos, object fieldValue)
```

Comment:
Can these methods be removed now?

Reply:
I do not think so, these override methods needs to be explicitly defined for Avro to serialize and de-serialize it.